eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -81,6 +81,26 @@ stages:
       excludeSdkContentTests: true       # ✅
       overrideDistroDisablingSha1: false # 🚫
       runOnline: true                    # ✅
+      withPreviousSDK: false             # 🚫
+
+  # Temporarily move this to PR build just to test the N-1 CI
+  - template: ../jobs/vmr-build.yml
+    parameters:
+      buildName: CentOSStream8_WithPreviousSDK
```

Comment:
I think there's a need to get crisper on our leg names especially after this additional leg.  For example this name doesn't properly differentiate itself with `Fedora36_Offline_Using_Previous`.

Here are some suggestion groupings

```
Stage1_CentOSStream8_Online
Stage1_CentOSStream8_Offline_N-1SDK
Stage1_Fedora36_Offline
Stage2_Fedora36_Offline
```

```
CentOSStream8_Online
CentOSStream8_Offline_N-1_SDK
Fedora36_Offline
Fedora36_Offline_Rebuild or Fedora36_Offline_N_SDK
```

Reply:
More ideas after talking with some source-build folks

```
CentOSStream8_Online_MsftSdk
Fedora36_Offline_MsftSdk
Fedora36_Offline_CurrentSourceBuiltSdk
CentOSStream8_Offline_PreviousSourceBuiltSdk
```

```
CentOSStream8_Online_MsftSdk_Stage1
Fedora36_Offline_MsftSdk_Stage1
Fedora36_Offline_Stage1Sdk_Stage2 
CentOSStream8_Offline_PreviousSourceBuiltSdk_Stage1
```

Reply:
Just a note - we'll need to update the release pipeline if these change

Reply:
@MichaelSimons we will now have to synchronize the list of legs between 3 places:
- VMR build
- SDK diff pipeline
- Release pipeline/checklist (if it needs to download the prereqs from a certain leg)

Reply:
I imagine this might become a point of friction